docs/docs/forms.md

```diff
@@ -335,6 +335,47 @@ For example:
 ```
 will return `undefined` if the field is empty.
 
+### Custom Input Fields
+
+You can create a custom field that integrates with Redwood through the use of `useRegister` and `useErrorStyles`. Each of these serving a different purpose depending on what you are trying to build.
```

Comment:
Could you please link `useRegister` and `useErrorStyles` to r-h-f docs?

Reply:
These are built into Redwood and have no equivalent in react-hook-form. `useRegister` is a wrapper around `register` which I did link to below. Let me know if that is okay?